src/app/identity.service.ts

```diff
@@ -81,19 +81,24 @@ export class IdentityService {
       person);
   }
 
-  get(id: string, jwt: string): Observable<Person> {
+  get(id: string, jwt: string, withTipBalances: boolean): Observable<Person> {
```

Comment:
It's sort of OK in the IDE, but when we read code that calls this in github we'd have to remember what the third param means - if we just see false or true that doesn't tell us much. So could we make it
```suggestion
  get(id: string, jwt: string, {withTipBalances}: {withTipBalances: boolean}): Observable<Person> {
```

The repetition of `withTipBalances` is a bit ugly, but then the calling code gets a lot clearer:
```diff
-    this.identityService.get(id, jwt, false).subscribe(
+    this.identityService.get(id, jwt, {withTipBalances: false}).subscribe(
```

Reply:
Or we could make false the default and then this PR doesn't need to change any of the callers except the one that passes true:
```suggestion
  get(id: string, jwt: string, {withTipBalances = false}: {withTipBalances?: boolean} = {}): Observable<Person> {
```